Assets/MixedRealityToolkit/Interfaces/Registrars/IMixedRealityServiceRegistrar.cs

```diff
@@ -85,13 +85,15 @@ public interface IMixedRealityServiceRegistrar
         /// </summary>
         /// <typeparam name="T">The interface type of the data provider to be registered.</typeparam>
         /// <param name="dataProviderInstance">An instance of the data provider to be registered.</param>
+        [Obsolete]
```

Comment:
You can remove this and related methods. Now is the time to make the last of the breaking changes and get this interface right. :)

Reply:
Definitely happy to remove :)